crates/teloxide-core/src/types/chat_member.rs

```diff
@@ -618,30 +632,6 @@ impl ChatMemberKind {
         }
     }
 
-    /// Returns `true` if the user is allowed to send audios, documents, photos,
-    /// videos, video notes and voice notes.
-    ///
-    /// I.e. returns **`false`** if the user
-    /// - has left or has been banned in the chat
-    /// - is restricted and doesn't have the [`can_send_media_messages`] right
-    /// Returns `true` otherwise.
-    ///
-    /// [`can_send_media_messages`]: Restricted::can_send_media_messages
-    #[deprecated(
-        since = "0.9.0",
-        note = "Match manually and use `can_send_media_messages` field directly. Details: https://github.com/teloxide/teloxide/issues/781"
-    )]
-    #[must_use]
-    pub fn can_send_media_messages(&self) -> bool {
```

Comment:
please add this function back and unbepricate it

Reply:
> and unbepricate it

Do you mean remove the `deprecated` attr?

Reply:
Yes; undeprecate*